src/components/settings/ThemePreferences.svelte

```diff
@@ -0,0 +1,60 @@
+<script lang="ts">
+  import { themeStore } from '$src/stores'
+  import { storeTheme, type Theme } from '$lib/theme'
+
+  const options = [
+    // {
```

Comment:
Please see the PR description regarding the commented out code in this file. Would love to chat about the best approach for this system default settings 👌🏼

Reply:
My understanding of how this worked originally was:

- On an initial visit, we set the theme to the system default
- Once the user toggles the theme, we ignore the system default

The idea being that the user has intent to change the theme, and we may not want to switch to the system default. Is the goal here to switch on UI toggle and when the system default changes?

Reply:
Yep, that's still how it works, but the designs have a `System Default` option in this radio selector. It gets tricky because if we allow the user to switch back to the system default, it will resolve to either `light` or `dark` anyway, so the correct radio option won't be selected. It may be easier to screenshare it, but I think the best option may be to not list the System Default option since it introduces quite a bit of complexity that won't necessarily benefit devs who are using the template

Reply:
@bgins I ended up removing the System Default option 🚀